### PR TITLE
EZP-28495: As Editor I want UI for external links management

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: https://jira.ez.no/browse/EZP-28495
> **Requires:  https://github.com/ezsystems/repository-forms/pull/185 and https://github.com/ezsystems/ezpublish-kernel/pull/2183** 

## Description

This PR contains UI implementation for link management. 

## Screenshots

screenshot-2017-12-14 link manager - ez platform ui

screenshot-2017-12-14 http facebook com - ez platform ui

screenshot-2017-12-14 editing url http facebook com - ez platform ui

### CODE REVIEW

File: Resources/public/css/views/linkmanager.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.ez-linkmanager-filter-searchquery {
+    float: left;
+    width: 50%;
+    text-align: left;
+}
+
+.ez-linkmanager-filter-searchquery input[type="search"] {
+    line-height: 2.2em !important;
```
Comment (dew326) [inline Resources/public/css/views/linkmanager.css]: Please avoid using `!important`, try stronger selector

**Comment by adamwojs** [comment]
PR updated according to @dew326 suggestions.

File: Resources/public/css/theme/views/pagination.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.ez-pagination {
+    text-align: center;
+}
+
+.ez-pagination span,
```
Comment (sunpietro) [inline Resources/public/css/theme/views/pagination.css]: Please, use classes instead. Avoid using generic element selectors.

File: Resources/public/css/views/linkmanager.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+.ez-linkmanager-filters {
+    clear: both;
+}
+
+.ez-linkmanager-filter-search {
+    float: left;
+    width: 50%;
+}
+
+.ez-linkmanager-filter-status {
```
Comment (sunpietro) [inline Resources/public/css/views/linkmanager.css]: You are using `float`. Do you use any kind of clearfix as well?

File: Resources/public/css/views/linkmanager.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    float: right;
+    width: 50%;
+    text-align: right;
+}
+
+#ez-linkmanager-filter-searchquery {
+    line-height: 2.2em;
```
Comment (sunpietro) [inline Resources/public/css/views/linkmanager.css]: Please, use either `px` or `rem` instead.

File: Resources/public/js/views/services/ez-navigationhubviewservice.js [hunk 5]
```diff
@@ -497,6 +497,10 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
                             this.get('rootMediaLocation').get('id'),
                             this.get('rootMediaLocation').get('contentInfo').get('mainLanguageCode')
                         ),
+                        this._getNavigationItem(
+                            Y.eZ.trans('navigationhub.linkmanager.list', {}, 'navigationhub'), "navigationhub",
+                            "adminGenericRoute", {uri: "pjax/link-management"}
```
Comment (sunpietro) [inline Resources/public/js/views/services/ez-navigationhubviewservice.js]: Please, use single quotes.

**Comment by adamwojs** [comment]
PR updated according to @sunpietro suggestions.

File: Resources/public/css/views/linkmanager.css (hunk 3 above)
Comment (adamwojs) [reply Resources/public/css/views/linkmanager.css]: I've added `clear: both` for `.ez-linkmanager-link-list` and I hope it's OK

Comment (sunpietro) [reply Resources/public/css/views/linkmanager.css]: 👍

**Pull request merged**